docs/source/commands/init.mdx

```diff
@@ -16,13 +16,64 @@ Running `rover init` starts a short wizard that helps you create a new graph.
 rover init
 ```
 
+<ExpansionPanel title="Experiencing issues?">
+
+    - If you experience a crash, run `rover config clear` and then run `rover config auth` before running `rover init` again.
+
+    - If you receive `error: Invalid API key found.` unset the `APOLLO_KEY` environment variable:
+
+        <Tabs>
+            <Tab label="Linux / MacOS">
+
+            ```terminal showLineNumbers=false
+            unset APOLLO_KEY
+            ```
+
+            </Tab>
+            <Tab label="Windows">
+            #### Powershell
+
+            ```terminal
+            Remove-Item Env:APOLLO_KEY
+            ```
+
+            #### Command Prompt
+            ```terminal
+            set APOLLO_KEY=
+            ```
+
+            </Tab>
+        </Tabs>
+
+     Then, run `rover config auth` before running `rover init` again.
+
+</ExpansionPanel>
+
 The wizard walks you through a set of questions to help you choose the best option for your use case. When you're done, you'll have:
 
 - A new graph in GraphOS
 - A set of local configuration files and boilerplate code that represent the graph 
 - Credentials to interact with GraphOS
     - These credentials are necessary for actions like [developing your graph locally](/rover/commands/dev) or [publishing graph updates](/graphos/platform/schema-management/delivery/publish).
 
+#### Options
+
+You can also pass options directly to `rover init` to skip the wizard in automated environments or scripts:
+
+```terminal showLineNumbers=false
+rover init --project-type <PROJECT_TYPE> --organization <ORGANIZATION> --project-use-case <PROJECT_USE_CASE> --project-name <PROJECT_NAME> --graph-id <GRAPH_ID>
+```
+
+**Available options:**
+
+| Option | Description | Possible Values |
+| ------ | ----------- | --------------- |
+| `--project-type` | Whether to create a new graph or add a subgraph to an existing graph. | `create-new`, `add-subgraph` |
```

Comment:
I'm not 100% aware of the convention in docs, but I'm wondering if we need to provide more context about where an `org ID` would be located just for a new user (who just created their org in Studio UI)?

Reply:
Good call! Rather than bloating this table, going to link out to the [GraphOS org page](https://www.apollographql.com/docs/graphos/platform/access-management/org) and add information about where to find your org information including ID.